data/battle_frontier_1.s

```diff
@@ -47,124 +88,377 @@ gUnknown_0860D068:: @ 860D068
+@ replacing .incbin "baserom.gba", 0x0060d090, 0x5c
+.4byte sub_818E9CC
+.4byte sub_818EA84
+.4byte sub_818ED28
+.4byte sub_818F9B0
+.4byte sub_818F9E0
+.4byte sub_818FA20
+.4byte sub_81903B8
+.4byte sub_81938A4
+.4byte sub_81938E0
+.4byte sub_8190298
+.4byte sub_81902E4
+.4byte sub_8193D40
+.4byte sub_8193D7C
+.4byte sub_81902F8
+.4byte sub_819033C
+.4byte sub_8194D48
+.4byte sub_8194D68
+.4byte sub_8194E44
+.4byte sub_8194EB4
+.4byte sub_8194EC0
+.4byte sub_8194EF8
+.4byte sub_8194F58
+.4byte sub_818F02C
 
 gUnknown_0860D0EC:: @ 860D0EC
-	.incbin "baserom.gba", 0x60d0ec, 0x10
+@ replacing .incbin "baserom.gba", 0x0060d0ec, 0x10
+.4byte 0x00000004, 0x00000008, 0x00100000, 0x00200000
 
 gUnknown_0860D0FC:: @ 860D0FC
-	.incbin "baserom.gba", 0x60d0fc, 0x10
+@ replacing .incbin "baserom.gba", 0x0060d0fc, 0x10
+.4byte 0xfffffffb, 0xfffffff7, 0xffefffff, 0xffdfffff
 
 gUnknown_0860D10C:: @ 860D10C
-	.incbin "baserom.gba", 0x60d10c, 0x40
+@ replacing .incbin "baserom.gba", 0x0060d10c, 0x40
+.byte 0x08, 0x00, 0x04, 0x08, 0x09, 0x0c, 0x08, 0x00, 0x0a, 0x08, 0x0c, 0x00, 0x0b, 0x04, 0x00, 0x08, 0x0c, 0x00, 0x04, 0x08, 0x0d, 0x0c, 0x08, 0x00, 0x0e, 0x08, 0x0c, 0x00, 0x0f, 0x04, 0x00, 0x08
+.byte 0x00, 0x00, 0x04, 0x08, 0x01, 0x0c, 0x08, 0x00, 0x02, 0x08, 0x0c, 0x00, 0x03, 0x04, 0x00, 0x08, 0x04, 0x00, 0x04, 0x08, 0x05, 0x0c, 0x08, 0x00, 0x06, 0x08, 0x0c, 0x00, 0x07, 0x04, 0x00, 0x08
 
 gUnknown_0860D14C:: @ 860D14C
-	.incbin "baserom.gba", 0x60d14c, 0x10
+@ replacing .incbin "baserom.gba", 0x0060d14c, 0x10
+.byte 0x00, 0x08, 0x04, 0x0c, 0x07, 0x0f, 0x03, 0x0b, 0x02, 0x0a, 0x06, 0x0e, 0x05, 0x0d, 0x01, 0x09
 
 gUnknown_0860D15C:: @ 860D15C
-	.incbin "baserom.gba", 0x60d15c, 0x40
+@ replacing .incbin "baserom.gba", 0x0060d15c, 0x40
+.byte 0x00, 0x08, 0x0c, 0x0e, 0x00, 0x08, 0x0c, 0x0e, 0x01, 0x08, 0x0c, 0x0e, 0x01, 0x08, 0x0c, 0x0e, 0x02, 0x09, 0x0c, 0x0e, 0x02, 0x09, 0x0c, 0x0e, 0x03, 0x09, 0x0c, 0x0e, 0x03, 0x09, 0x0c, 0x0e
+.byte 0x04, 0x0a, 0x0d, 0x0e, 0x04, 0x0a, 0x0d, 0x0e, 0x05, 0x0a, 0x0d, 0x0e, 0x05, 0x0a, 0x0d, 0x0e, 0x06, 0x0b, 0x0d, 0x0e, 0x06, 0x0b, 0x0d, 0x0e, 0x07, 0x0b, 0x0d, 0x0e, 0x07, 0x0b, 0x0d, 0x0e
 
 gUnknown_0860D19C:: @ 860D19C
-	.incbin "baserom.gba", 0x60d19c, 0x4
+@ replacing .incbin "baserom.gba", 0x0060d19c, 0x4
+.byte 0x17, 0x1b, 0x1d, 0x1e
 
 gUnknown_0860D1A0:: @ 860D1A0
-	.incbin "baserom.gba", 0x60d1a0, 0x20
+@ replacing .incbin "baserom.gba", 0x0060d1a0, 0x20
+.byte 0x10, 0x18, 0x1c, 0x1e, 0x11, 0x18, 0x1c, 0x1e, 0x12, 0x19, 0x1c, 0x1e, 0x13, 0x19, 0x1c, 0x1e, 0x14, 0x1a, 0x1d, 0x1e, 0x15, 0x1a, 0x1d, 0x1e, 0x16, 0x1b, 0x1d, 0x1e, 0x17, 0x1b, 0x1d, 0x1e
 
 gUnknown_0860D1C0:: @ 860D1C0
-	.incbin "baserom.gba", 0x60d1c0, 0x10
+@ replacing .incbin "baserom.gba", 0x0060d1c0, 0x10
+.byte 0x00, 0x0f, 0x08, 0x07, 0x03, 0x0c, 0x0b, 0x04, 0x01, 0x0e, 0x09, 0x06, 0x02, 0x0d, 0x0a, 0x05
 
 gUnknown_0860D1D0:: @ 860D1D0
-	.incbin "baserom.gba", 0x60d1d0, 0x44
+@ replacing .incbin "baserom.gba", 0x0060d1d0, 0x44
+.4byte 0x82c3188
```

Comment:
Don't leave pointers in dumped data, ever. Always make sure they point to a symbol (that's kind of the reason dumping is so important).